index.js

```diff
@@ -6,15 +7,258 @@ function ask(questionText) {
+  if (blueprintQ === 'leave') {
+    console.log('\n***You place the blueprint back down***')
+  } else {
+    console.log('\nOh no, the poison left on the blueprint spreads throughout your body\nyou have now died\nINCONCEIVABLE! ')
+    process.exit()
+  }
+
+
+  console.log((roomsLookUp[player.location].desc))
+  //---------------- Main Story code: move, take, drop, status, inventory ----------
+  while (true) {
+    let room = roomsLookUp[player.location];
+    let question = await ask('move, drink, take, drop, check inventory, read, status\nWhat do you want to do? ')
+    if (question === 'move') {
+      console.log(shipRooms[player.location].canChangeTo)
+      let clarifyingQ = await ask('\nWhat room do you want to go to? ')
+      if (clarifyingQ === 'engineBlock' && engineBlock.locked === true) {
+        await passcodeEngineBlock()
+      }
+      if (clarifyingQ === 'hangarControlRoom') {
+        turnPowerOnInHangarControlRoom()   
+      }
+      room.moveBetweenRooms(clarifyingQ)
+      room = roomsLookUp[player.location]
+      console.log(room.desc)
+    }
+    else if (question === 'take') {
+      let takeQ = await ask('\nWhat would you like to take? ')
+      if (room.inv.includes(takeQ)) {
+        console.log('\nYou have taken this item for your inventory.')
+        player.inventory.push(takeQ)
+      } else {
+        console.log('That specific item is not found in this room. Look elsewhere. ')
+      }
+    }
+    else if (question === 'drop') {
+      let dropQ = await ask('\nWhat would you like to drop? ')
+      if (player.inventory.includes(dropQ)) {
+        console.log('\n\nYou have dropped this item. ')
+        player.inventory.pop[dropQ]
+      } else {
+        console.log('\nThat specific item is not found in your inventory. ')
+      }
+    }
+    else if (question === 'check inventory') {
+      let inventoryQ = await ask('\nWould you like to check your inventory? ')
+      if (inventoryQ === 'yes') {
+        console.log('The items you have in your inventory are: ' + player.inventory)
+      } else {
+        console.log('\n\nOK ... moving on ')
+      }
+    }
+    else if (question === 'status') {
+      let statusQ = await ask('\nWould you like to check your status? ')
+      if (statusQ === 'yes') {
+        console.log('You are currently feeling: ' + player.status)
+      } else {
+        console.log('\n\nOK ... moving on ')
+      }
+    }
+    else if (question === 'drink') {
+      let drinkQ = await ask('\nWould you like to drink this? ')
+      if (drinkQ === 'yes' && player.inventory.includes['energyDrink']) {
+        console.log(player.status)
+        player['changeStatus']()
+        console.log('\n\nYou are currently feeling: ' + player.status)
+      } else {
+        console.log('\n\nYou don\'t have anything to drink ')
+      }
+    }
```

Comment:
This doesn't entirely work. You need to reference your item lookup table so that the user input becomes the item of the energy drink rather then the string